src/main/java/uk/gov/hmcts/cmc/claimstore/services/DocumentsService.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.documents.CountyCourtJudgmentPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.DefendantResponseCopyService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@Service
+public class DocumentsService {
+    private final SealedClaimDocumentService sealedClaimDocumentService;
+    private final ClaimService claimService;
+    private final DefendantResponseCopyService defendantResponseCopyService;
+    private final CountyCourtJudgmentPdfService countyCourtJudgmentPdfService;
+
+    @Autowired
+    public DocumentsService(final SealedClaimDocumentService sealedClaimDocumentService,
```

Comment:
This is not how we format constructors, see a class below from the same PR https://github.com/hmcts/cmc-claim-store/pull/97/files#diff-634273b78a3695859749686ad777963e